gc_registry/tests/certificate/test_certificate_services.py

```diff
@@ -239,6 +239,15 @@ def test_transfer_gcs(
         Transfer a fixed number of certificates from one account to another.
         """
 
+        # Whitelist the source account for the target account
+        fake_db_account_2 = db_write_session.merge(fake_db_account_2)
```

Comment:
Nice - do we have test case where not whitelisted to check it fails?

Reply:
Can add that case